### DISCUSSION TITLE
Housekeeping Genes?

### DISCUSSION THREAD

**Discussion by coopershawna** [stuart-lab/signac#871] [Q&A] [answered]
Does anyone check housekeeping genes like we typically do for qPCR and western blotting?  I just did and I'm trying to decide how I feel about my results.  
Some of them have genes on the opposite strand and I wonder how much that is contributing to peaks and cicero scores?

Thoughts?

Thanks!

**Comment by timoast** [comment, marked as answer]
For a set of positive control regions in scATAC-seq, we typically look at the accessibility around TSSs, and this is captured by the TSS enrichment score.

I don't think that the strand the gene is on should have any impact of peak calling/quantification or the cicero method.
